helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/templates/liveindexing-config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "alfresco-elasticsearch-connector.fullName" . }}-configmap
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    component: search
+data:
+  ELASTICSEARCH_INDEXNAME: "{{ .Values.indexName }}"
+  SPRING_ELASTICSEARCH_REST_URIS: http://elasticsearch-master:9200
```

Comment:
This might need to be set to some other value, or we need an external service resource matching that name.